otlp/traces_test.go

```diff
@@ -137,8 +137,8 @@ func TestTranslateLegacyGrpcTraceRequest(t *testing.T) {
 	assert.Equal(t, "resource_attr_val", ev.Attributes["resource_attr"])
 	assert.Equal(t, 1, ev.Attributes["span.num_links"])
 	assert.Equal(t, 1, ev.Attributes["span.num_events"])
-	assert.Equal(t, "instr_scope_name", ev.Attributes["instrumentation_scope.name"])
```

Comment:
Is `instrumentation_scope` --> `library` a breaking change we need to worry about?